src/css/dialog.scss

```diff
@@ -21,7 +21,11 @@
   #chr,
   #matrix,
   #network,
-  #mapping-file {
+  #mapping-file,
```

Comment:
This css is in bad shape. Since we are having many ids with width=220px, it makes more sense to css-select them with a class, or a general tag.